de.dlr.sc.virsat.cef.docs.feature/docs/VirSat_CEF_ReleaseNotes.txt

```diff
@@ -3,6 +3,12 @@ VirSat CEF Release Notes
 
 VirSat CEF is based on VirSat Core. Please don't forget to read the Release Notes of VirSat Core as well!
 
+Release 4.9.2 (21st November 2019)
+----------------------------------
+
+Fixed Bugs:
+	Fixed crash on opening multiple instances of VirSat (in Core 4.9.2)
```

Comment:
I think it does not necessarily be mentioned here. It is mentioned in the release notes of core.

Reply:
I agree! Let's not do this anymore. I did it because CEF release notes already mention a lot of Core stuff.